src/Microsoft.AspNetCore.Http.Features/Authentication/SignInContext.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Microsoft.AspNetCore.Http.Features.Authentication
 {
+    [Obsolete("See https://go.microsoft.com/fwlink/?linkid=845470")]
```

Comment:
When I smush all the changes together for the merge, I won't check in anything outside of the 2 new assemblies, so you don't need to review anything outside of the Authentication packages in this PR